### PR TITLE
{bio}[GCC/11.2.0] angsd v0.940

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3141.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ee5194ff3a783c64e51e39907d2af89a for a full test report.

File: easybuild/easyconfigs/a/angsd/angsd-0.940-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      'misc/NGSadmix', 'misc/contamination2', 'misc/haploToPlink', 'misc/ngsPSMC',
+      'misc/ibs'], 'bin'),
+    'doc',
+]
+
+sanity_check_paths = {
+    'files': ['bin/angsd'],
+    'dirs': ['doc'],
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/a/angsd/angsd-0.940-GCC-11.2.0.eb:37]: @boegel could we add a (few?) sanity check command(s)?

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/31846a466e1857cea288ce07d43c8c44 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen2

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Pull request merged**